server/.eslintrc

```diff
@@ -8,4 +8,4 @@
   "rules": {
     "no-console": "off"
   }
-}
+}
```

Comment:
Why're we removing the newline at the end of this file?

Reply:
Not sure. I didn't even think about it when I pasted it back in. I can put it back in.